### PR TITLE
Test Resource

### PR DESCRIPTION
**Pull request by nateude**
test resource will log data from website tests 

field | type
------------ | -------------
id | integer
status | string
wpt_id | string
url | string
load_time | integer
first_byte | integer
start_render | integer
speed_index | integer
dom_elements | integer
complete_time | integer
complete_requests | integer
complete_bytes_in | integer
loaded_time | integer
loaded_requests | integer
loaded_bytes_in | integer
loaded_certificates | integer

### CODE REVIEW

**Comment by plasticwrapper** [comment]
Are these naming conventions standard WPO API ones?

File: app/controllers/tests_controller.rb [hunk 1]
```diff
@@ -0,0 +1,4 @@
+class TestsController < ApplicationController
```
Comment (plasticwrapper) [inline app/controllers/tests_controller.rb]: Oh wow. Something called a test under the app directory?  My mind is hurting right now. Tests and app code are different. This is a production controller named "test". Hmmm can we name them performance runs? Or something less ambiguous that means something else in the rails community?

Comment (plasticwrapper) [inline app/controllers/tests_controller.rb]: That *doesnt mean something else.

**Comment by plasticwrapper** [comment]
Maybe profiles

**Comment by plasticwrapper** [comment]
Other than that naming issue, looks good so far dude!  Thanks for setting this up!

**Comment by nateude** [comment]
@plasticwrapper That is fair feedback. 

The plan in my head is to have three sets of data. One that is the website settings, one that is the data and one that is test instructions. So how about Websites, Profiles (test instructions), and Reports (Test Data) ?

**Comment by plasticwrapper** [comment]
@nateude sounds good to me.  Great names!  I like them.

**Pull request closed without merging**